scripts/add-kind-cluster.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+set -o errexit
+
+assert_command() {
+  local command=$1
+
+  printf "Checking for presense of $command..."
+  local path_to_executable=$(command -v ${command})
+
+  if [ -z "$path_to_executable" ] ; then
+    echo -e "Cannot find ${command} executable. Is it on your \$PATH?"
+    exit 1
+  fi
+  printf "OK\n"
+}
+
+for COMMAND in "kind" "docker" "kubectl"; do
+    assert_command "${COMMAND}"
+done
+
+# create registry container unless it already exists
+reg_name='kind-registry'
+reg_port='5000'
+running="$(docker inspect -f '{{.State.Running}}' "${reg_name}" 2>/dev/null || true)"
+if [ "${running}" != 'true' ]; then
+  docker run \
+    -d --restart=always -p "127.0.0.1:${reg_port}:5000" --name "${reg_name}" \
+    registry:2
+fi
+
+# create a cluster with the local registry enabled in containerd
+cat <<EOF | kind create cluster --name dscp-flux-infra --config=-
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+containerdConfigPatches:
+- |-
+  [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
+    endpoint = ["http://${reg_name}:${reg_port}"]
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
+  extraPortMappings:
+  - containerPort: 80
+    hostPort: 3080
+    protocol: tcp
+  - containerPort: 443
+    hostPort: 3443
+    protocol: tcp
```

Comment:
We should forward the node ports here so we can access the node from the polkadot gui